tests/console/verify_efi_mok.pm

```diff
@@ -185,7 +185,7 @@ sub run {
     # run fs check on ESP
     record_info "ESP", "Partition [$esp_details->{partition}], \nFilesystem [$esp_details->{fs}],\nMountPoint [$esp_details->{mount}]";
     assert_script_run "umount $esp_details->{mount}";
-    assert_script_run "fsck.vfat -tvV $esp_details->{partition}";
+    assert_script_run "fsck.vfat -tvVp $esp_details->{partition}";
```

Comment:
Why `-p`? Shouldn't the test verify that it's clean?

Reply:
It failed with https://openqa.suse.de/tests/6864292#step/verify_efi_mok/48

Reply:
`-t  Mark unreadable clusters as bad.`

sounds like it shouldn't be used here (and won't really make sense on virtual HW anyway)

Reply:
https://openqa.suse.de/tests/6869789#step/verify_efi_mok/48